test/distributed/_composable/fsdp/test_fully_shard_init.py

```diff
@@ -0,0 +1,45 @@
+# Owner(s): ["oncall: distributed"]
+
+import unittest
+
+import torch
+
+from torch.distributed._composable.fsdp._fsdp_init import _normalize_device
+from torch.testing._internal.common_cuda import TEST_CUDA
+from torch.testing._internal.common_fsdp import FSDPTestMultiThread
+from torch.testing._internal.common_utils import run_tests
+
+
+class TestFullyShardDeviceArg(FSDPTestMultiThread):
+    """Tests the ``device`` argument."""
+
+    @property
+    def world_size(self) -> int:
+        return 1
+
+    def test_normalize_device_cpu(self):
+        for device in ("cpu", torch.device("cpu")):
+            self.assertEqual(_normalize_device(device), torch.device("cpu"))
+
+    def test_normalize_device_meta(self):
```

Comment:
nit: `cpu` `meta` looks exactly same logic, maybe we can put them in the same test case?